lib/bezier.js

```diff
@@ -795,6 +795,12 @@
           np3 = this.get(e);
 
           arc = utils.getccenter(np1, np2, np3);
+          
+          //also save the t values
+          arc.t_s = s;
```

Comment:
it's a non-linear relation, so while I can see a use for start and end, not sure middle would contribute anything. I'd suggest naming them explicitly then:
```
arc.interval = {
  start: s,
  end: e
}
```